src/hooks/stores/preferences.ts

```diff
@@ -46,6 +65,7 @@ export class Preferences implements IPreferences {
         settingsDir: this.settingsDir,
         onboarding: this.onboarding,
         darkMode: this.darkMode,
+        fonts: this.fonts,
```

Comment:
Create defaults for all of the font variables in the settings store (see client/preferences)